datadog_checks_dev/datadog_checks/dev/tooling/commands/release/make.py

```diff
@@ -87,7 +87,7 @@ def make(ctx, checks, version, end, initial_release, skip_sign, sign_only, exclu
         abort('Please create a release branch, you do not want to commit to master directly.')
 
     # Signing is done by a pipeline in a separate commit
-    if not core_workflow and not sign_only:
+    if not core_workflow and not sign_only or checks == ["ddev"]:
```

Comment:
so if we release `ddev` **and** `datadog_checks_dev` in one go, this won't trigger?

Reply:
hmm, where do we exclude `datadog_checks_dev` from signing? Would it make sense to exclude `ddev` in the same place?

Reply:
We don't since I said we DO sign it

Reply:
I get it now, sorry about the confusion.